src/planscape/plan/views.py

```diff
@@ -315,6 +318,37 @@ def get_project(request: HttpRequest) -> HttpResponse:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
 
 
+@csrf_exempt
+def delete_projects(request: HttpRequest) -> HttpResponse:
+    try:
+        # Check that the user is logged in.
+        owner = _get_user(request)
+
+        body = json.loads(request.body)
+        project_ids = body.get('project_ids', None)
+        if project_ids is None or not (isinstance(project_ids, list)):
+            raise ValueError("Must specify project_ids as a list")
+
+        projects = [Project.objects.get(id=project_id) for project_id in project_ids]
+        
+        # Check that the user owns the projects
```

Comment:
we can alternatively add owner=user in the .get() filters above. the downside of doing that is that we won't return the error below

Reply:
Returning a specific error sounds more user-friendly to me